src/app-modules/docker-compose

```diff
@@ -13,8 +13,6 @@
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
 
-set -x
-
```

Comment:
this one was for debug purposes, since I see we are moving closer to production I am ok with removing.